Github-copilot-code-review-best-practices.html

```
@@ -0,0 +1,2666 @@
+        footer.page-footer {
+          padding-left: 1.5rem;
+          padding-right: 1.5rem;
+        }
+        .hero {
+          padding-top: 4.4rem;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <button class="sidebar-toggle" id="sidebarToggle" aria-label="目次を開閉">☰</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)